### PR TITLE
Improved Issue Templates

### PR DESCRIPTION
**Pull request by AnishDe12020**
## Changes proposed

- [x] Improve bug report template
- [x] Improve feature request template
- [x] Improve submission request template  

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

I have commented out descriptions in issue templates.

Also, a submission request issue label can be added and automatically added to any submission request issue (edit frontmatter). Person with perms to edit repo content needs to do this.

### CODE REVIEW

**Review by MaheshtheDev — APPROVED** [review]
Great Addition @AnishDe12020

File: .github/ISSUE_TEMPLATE/submission_request.md [hunk 1]
```diff
@@ -6,28 +6,27 @@ labels: ''
 assignees: ''
 ---
 
-**Note**: Please read [this article](https://docs.genicsblog.com/author/create-a-submission-request) before continuing further.
+<!-- **Note**: Please read [this article](https://docs.genicsblog.com/author/create-a-submission-request) before continuing further. -->
```
Comment (gouravkhunger) [inline .github/ISSUE_TEMPLATE/submission_request.md:9]: We can remove the markdown in the comment block and add normal text referring to the link. Example: `<!-- Please read this document before continuing further: https://docs.genicsblog.com/author/create-a-submission-request -->`

File: .github/ISSUE_TEMPLATE/submission_request.md [hunk 2]
```diff
@@ -6,28 +6,27 @@ labels: ''
...
 
 ### Idea
-
-In this space, explain what your article is about.
+<!-- In this space, explain what your article is about. -->
 
 ### Topics
 
-Checkout the [existing topics](https://genicsblog.com/topics).
+<!-- Checkout the [existing topics](https://genicsblog.com/topics). -->
```
Comment (gouravkhunger) [inline .github/ISSUE_TEMPLATE/submission_request.md:16]: same here, let's have just a plain link

File: .github/ISSUE_TEMPLATE/submission_request.md [hunk 3]
```diff
@@ -6,28 +6,27 @@ labels: ''
...
-You must be the original author of that article to re-post at Genics Blog.
+You must be the original author of that article to re-post at Genics Blog. -->
 
 ### Remarks
 
-Please add anything you would like to share.
+<!-- Please add anything you would like to share. -->
 
-**Note:** Please remove all the unnecessary lines and comments before submitting the issue.
+<!-- **Note:** Please remove all the unnecessary lines and comments before submitting the issue. -->
```
Comment (gouravkhunger) [inline .github/ISSUE_TEMPLATE/submission_request.md:32]: This too.

**Review by gouravkhunger — CHANGES_REQUESTED** [review]
Except these few fixes, pretty good work! Thank you :)

File: .github/ISSUE_TEMPLATE/submission_request.md (hunk 3 above)
Comment (AnishDe12020) [reply .github/ISSUE_TEMPLATE/submission_request.md:32]: I didn't get it here, there is no link

Comment (gouravkhunger) [reply .github/ISSUE_TEMPLATE/submission_request.md:32]: I mean the `**Note**` markdown could be removed 😄

Comment (AnishDe12020) [reply .github/ISSUE_TEMPLATE/submission_request.md:32]: Oh ok

Comment (AnishDe12020) [reply .github/ISSUE_TEMPLATE/submission_request.md]: Done

File: .github/ISSUE_TEMPLATE/submission_request.md (hunk 2 above)
Comment (AnishDe12020) [reply .github/ISSUE_TEMPLATE/submission_request.md]: Done

File: .github/ISSUE_TEMPLATE/submission_request.md (hunk 1 above)
Comment (AnishDe12020) [reply .github/ISSUE_TEMPLATE/submission_request.md]: Done

**Review by gouravkhunger — APPROVED** [review]

**Comment by gouravkhunger** [comment]
Thank you!

**Pull request merged**